app/services/journey_logger_service.rb

```diff
@@ -29,13 +29,15 @@ def build_attributes(calculation_result, check)
         capital_contribution: calculation_result.raw_capital_contribution&.positive? || false,
         income_contribution: calculation_result.raw_income_contribution&.positive? || false,
         asylum_support: check.asylum_support || false,
-        matter_type: matter_type(check),
+        matter_type: proceeding_type(check),
       }
     end
 
-    def matter_type(check)
-      if !check.controlled?
-        check.matter_type
+    def proceeding_type(check)
```

Comment:
I don't think we should call this proceeding type. "Proceeding type" means something quite specific, and is normally represented in the code by something like "DA001" or "IM030" etc. In this case, it is the matter type we are talking about. (There are multiple proceeding types associated with every matter type, I believe).